src/main/main.ts

```diff
@@ -34,8 +39,87 @@ export function importOnestopToBasecamp(): void {
     deleteOldRows(processedRowIds);
 }
 
+/**
+ * Processes existing event rows by checking whether the row has been modified and if so indiscriminately
+ * updates all the tasks with the new information
+ * 
+ * @param row 
+ */
 function processExistingRow(row: Row): void {
```

Comment:
this is a huge function 🥲 any way to break this up into smaller functions?

Reply:
Will do

Reply:
It's broken up now

Reply:
nice i like it, much cleaner